mrp_subcontracting_receipt_set_quantity/models/stock_move.py

```diff
@@ -12,5 +12,5 @@ def _set_quantities_to_reservation(self):
             if move.is_subcontract:
                 move = move.with_context(move_dest_qty=move.product_uom_qty)
                 self -= move
-                return super(StockMove, move)._set_quantities_to_reservation()
-        return super()._set_quantities_to_reservation()
+                super(StockMove, move)._set_quantities_to_reservation()
+        return super(StockMove, self)._set_quantities_to_reservation()
```

Comment:
To make the logic less misleading.
```suggestion
                super(StockMove, move)._set_quantities_to_reservation()
        super()._set_quantities_to_reservation()
        return
```